tools/fuzzer/lldb_vs_lldb_eval_libfuzzer_test.cc

```diff
@@ -32,6 +34,40 @@
 
 static fuzzer::LibfuzzerState g_state;
 
+static const char* kLoggingPrefix = "[lldb-eval-fuzzer] ";
+
+// How often to log statistics on UB (e.g. every 500 iterations)?
+static const int kUbStatsPeriod = 500;
+
+extern "C" const char* __lsan_default_options() {
+  return "print_suppressions=0";
+}
+
+extern "C" const char* __lsan_default_suppressions() {
```

Comment:
Maybe leave comment or a TODO with a short explanation why we need this and when we can get rid of it